### PR TITLE
Update developerguide.adoc

### PR DESCRIPTION
**Pull request by jdahlblom**

### CODE REVIEW

File: Scripts/DCS-BIOS/doc/developerguide.adoc [hunk 1]
```diff
@@ -19,7 +20,7 @@ It is part of the "Administratively Scoped IPv4 Multicast Space", which is to mu
 
 If you cannot or do not want to deal with multicast UDP, you can also make DCS-BIOS send a copy of the export data stream to a unicast address by editing `BIOSConfig.lua`.
 
-You can also use the `socat` utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
+You can also use the `socat` or `DCSBIOSBridge`  utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/developerguide.adoc:23]: Should we include a link to dcsbiosbridge?

File: Scripts/DCS-BIOS/doc/developerguide.adoc [hunk 2]
```diff
@@ -119,142 +120,55 @@ If you feed the export stream data you receive from DCS-BIOS to the `processChar
...
+lib/io:: Contains all files needed for TCP/UDP transmission.
 
-lib/Util.lua:: This file includes a lot of utility classes and functions.
-Most of them implement the `MemoryMap` class and related classes or provide shortcuts to quickly define controls in aircraft modules.
+Protocol.lua:: Writes modules to memory and JSON.
 
-lib/A-10C.lua, lib/UH-1H.lua, etc:: Each aircraft module has its own file where all of its controls are defined.
+Module.lua:: This is the main class which all aircraft modules inherits from. Contains all functions needed for creating input and export functions for a control.
+
+lib/modules/air_craft/A-10C.lua, lib/modules/air_craft/UH-1H.lua, etc:: Each aircraft module has its own file where all of its controls are defined.
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/developerguide.adoc:137]: aircraft, not air_craft

File: Scripts/DCS-BIOS/doc/developerguide.adoc [hunk 3]
```diff
@@ -119,142 +120,55 @@ If you feed the export stream data you receive from DCS-BIOS to the `processChar
...
 
 == Adding a New Aircraft Module
 
-This section will describe how to add support for another aircraft module.
+This section describes how to add support for another aircraft module.
 
 === Export Modules in DCS-BIOS
 
-DCS-BIOS consists of several export modules (they are what you select in the "module" drop-down field in the control reference documentation). Each export module is assigned to one or multiple aircraft and several export modules can be active at the same time.
+DCS-BIOS consists of several export modules (they are what you select in the "module" drop-down field in `Bort` or `BIOSBuddy`. Each export module is assigned to one or multiple aircraft and several export modules can be active at the same time.
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/developerguide.adoc:146]: Should we add links to these applications?

**Review by charliefoxtwo — DISMISSED** [review]
Minor suggestions and spelling, but otherwise looks good

File: Scripts/DCS-BIOS/doc/developerguide.adoc (hunk 1 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/developerguide.adoc]: Fixed

File: Scripts/DCS-BIOS/doc/developerguide.adoc (hunk 2 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/developerguide.adoc]: Fixed

File: Scripts/DCS-BIOS/doc/developerguide.adoc (hunk 3 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/developerguide.adoc]: Fixed

**Comment by jdahlblom** [comment]
I will check the user guide too in this PR.

File: Scripts/DCS-BIOS/doc/developerguide.adoc [hunk 4]
```diff
@@ -20,7 +20,7 @@ It is part of the "Administratively Scoped IPv4 Multicast Space", which is to mu
 
 If you cannot or do not want to deal with multicast UDP, you can also make DCS-BIOS send a copy of the export data stream to a unicast address by editing `BIOSConfig.lua`.
 
-You can also use the `socat` or `DCSBIOSBridge`  utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
+You can also use the `socat` or `DCSBIOSBridge` (https://github.com/DCS-Skunkworks/DCSBIOSBridge)  utility to connect a serial port (and just about anything else that can be read or written) to the DCS-BIOS import and export streams.
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/developerguide.adoc:23]: You can use markdown links like so [example](https://www.example.com)

```
[example](https://www.example.com)
```

**Review by charliefoxtwo — APPROVED** [review]

File: Scripts/DCS-BIOS/doc/developerguide.adoc (hunk 4 above)
Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/doc/developerguide.adoc:23]: And if you want the monospace text, you can do that too [`example`](https://www.example.com)  
```
[`example`](https://www.example.com) 
```

Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/developerguide.adoc:23]: OK will update in next PR. Working on the userguide.

**Pull request merged**